Xamarin.Forms.Platform.iOS/Renderers/FrameRenderer.cs

```diff
@@ -79,7 +79,13 @@ public virtual void SetupLayer()
 			if (Element.BackgroundColor == Color.Default)
 				_actualView.Layer.BackgroundColor = ColorExtensions.BackgroundColor.CGColor;
 			else
+			{
+				// BackgroundColor gets set on the base class too which messes with
+				// the corner radius, shadow, etc. so override that behaviour here
+				BackgroundColor = null;
```

Comment:
hmm, this was causing other tests to fail, though

Reply:
that's why i removed it!

Reply:
Oh, ok!. I changed it. The BackgroundColor should work but I removed this line.